docs/product/components/checkbox.html

```diff
@@ -10,7 +10,7 @@
 {% highlight html %}
 <!-- Base -->
 <fieldset>
-    <div class="d-flex gs8">
+    <div class="d-flex g8">
```

Comment:
## Change requested

Since `<label>` elements are rendered as `display: table`, the gap property doesn't apply. Any `<label>` with parents that include a gap property need either to have a margin added or have their `display` property altered to honor `gap`.